pkg/client-sdk/covenant_client.go

```diff
@@ -1695,7 +1693,10 @@ func (a *covenantArkClient) getOffchainBalance(
 	return balance, amountByExpiration, nil
 }
 
-func (a *covenantArkClient) getVtxos(ctx context.Context, opts *CoinSelectOptions) ([]client.Vtxo, error) {
+func (a *covenantArkClient) getVtxos(
+	ctx context.Context,
+	withExpiryCoinselect bool, opts *CoinSelectOptions,
```

Comment:
This is an internal getter, I think it's ok to have "breaking change" in it. Let's use `opts.WithExpirySorting` instead of duplicating the bool argument (moreover, withExpiryCoinselect is not used in that function).